Sources/ProjectSpec/ProjectSpec.swift

```diff
@@ -29,6 +29,7 @@ public struct ProjectSpec {
 
     public struct Options {
         public var carthageBuildPath: String?
+        public var mirrorsFileSystem: Bool
```

Comment:
As XcodeGen already somewhat mirrors the file system, can we call this `createIntermediateGroups` or something similar?